src/routes/feedback/components/Feedback.js

```diff
@@ -0,0 +1,216 @@
+}
+  from '../../../modules/endpoint/FeedbackEndpoint'
+import type { TFunction } from 'react-i18next'
+import CleanLink from '../../../modules/common/components/CleanLink'
+import type { FeedbackDataType } from '../../../modules/endpoint/FeedbackEndpoint'
+import { CATEGORIES_ROUTE } from '../../../modules/app/routes/categories'
+import { EVENTS_ROUTE } from '../../../modules/app/routes/events'
+import { EXTRAS_ROUTE } from '../../../modules/app/routes/extras'
+import { SEARCH_ROUTE } from '../../../modules/app/routes/search'
+import { DISCLAIMER_ROUTE } from '../../../modules/app/routes/disclaimer'
+
+const FeedbackBox = styled.div`
+  display: flex;
+  width: 400px;
+  height: auto;
+  box-sizing: border-box;
+  flex-direction: column;
+  justify-content: space-between;
+  padding: 20px;
+  border-radius: 10px;
+  border-color: #585858;
+  font-size: ${props => props.theme.fonts.contentFontSize};
+`
+
+const Header = styled.div`
+  display: flex;
+  width: 100%;
+  flex-direction: row;
+  justify-content: space-between;
+`
+
+const CloseButton = styled(CleanLink)`
+  font-size: 2rem;
+`
+
+const Title = styled.div`
+  padding: 15px 0 10px;
+  font-size: ${props => props.theme.fonts.subTitleFontSize};
+`
+
+const Description = styled.div`
+  padding: 10px 0 5px;
+`
+
+const CommentField = styled.textarea`
+  resize: none;
+`
+
+const SubmitButton = styled(CleanLink)`
+  margin: 15px 0;
+  padding: 5px;
+  background-color: ${props => props.theme.colors.themeColor};
+  color: ${props => props.theme.colors.backgroundAccentColor};
+  text-align: center;
+  border-radius: 0.25em;
+`
+
+export type FeedbackDropdownType = {
+  value: string,
+  feedbackType: string | null,
+  label: string
+}
+
+type PropsType = {
+  cities: Array<CityModel>,
+  city: string,
+  language: string,
+  id?: number,
+  title: string,
+  alias?: string,
+  query?: string,
+  route: string,
+  isPositiveRatingSelected: boolean,
+  pathname: string,
+  isOpen: boolean,
+  t: TFunction
+}
+
+type StateType = {
+  feedbackOptions: Array<FeedbackDropdownType>,
+  selectedFeedbackOption: FeedbackDropdownType,
+  comment: string
+}
+
+class Feedback extends React.Component<PropsType, StateType> {
+  constructor (props: PropsType) {
+    super(props)
+    const feedbackOptions = this.getFeedbackOptions()
+    this.state = {feedbackOptions: feedbackOptions, selectedFeedbackOption: feedbackOptions[0], comment: ''}
+  }
+
+  onCommentChanged = (event: {target: {value: string}}) => this.setState({comment: event.target.value})
+
+  onFeedbackOptionChanged = (selectedDropdown: FeedbackDropdownType) => {
+    this.setState({selectedFeedbackOption: selectedDropdown})
+  }
+
+  componentDidUpdate (prevProps: PropsType) {
+    const {isOpen} = this.props
+    const prevIsOpen = prevProps.isOpen
+
+    // If the Feedback is opened, we have to reset and initialize the state
+    if (prevIsOpen !== isOpen && isOpen) {
+      /* eslint-disable react/no-did-update-set-state */
```

Comment:
We have to find a solution if we want to change linting rules. I think its easier to find disabled linting rules if we collect them in the project eslint config, instead of having lines like this spread accross the project.

Also you can enable lines again after disabling them. So you only disable it for a selected portion of the code.

Reply:
This is just a temporary fix, I want to delete these rules entirely from our eslint config, but I still don't have permissions to push to the eslint project.